judo-runtime-core-jsl-itest/common/src/main/java/hu/blackbelt/judo/runtime/core/jsl/fixture/JudoDatasourceFixture.java

```diff
@@ -95,24 +103,38 @@ public void teardownDatasource() {
         }
     }
 
-    public void dropSchema() {
+    public void truncateTables(RdbmsModel rdbmsModel) {
+        RdbmsUtils rdbmsUtils = new RdbmsUtils(rdbmsModel.getResourceSet());
+        HashMap<String, String> statmentTemplate = new HashMap<>();
```

Comment:
This hasmap I think have to be static in the begining of class. You can us ImmutableMap for it.

Reply:
[bb3a87b](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/242/commits/bb3a87b315d5eff48012bff0e9765287ac0b52e8)